src/linter/rule.zig

```diff
@@ -20,11 +20,19 @@ pub const NodeWrapper = struct {
     }
 };
 
+pub const Line = struct {
```

Comment:
There's already a `Line` struct in `GraphicalFormatter`. I'll move it over to `span.zig`.